NetworkingFramework-main/TestFramework/HttpClient.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  HttpClient.swift
+//  TestFrameworkTests
+//
+//  Created by Дмитрий Игнатьев on 27.11.2021.
+//
+
+import UIKit
+
```

Comment:
Usually we make only one newline between import and protocols/classes/enums/structs definitions

Reply:
Done